shared/subprocess/manager.go

```diff
@@ -4,19 +4,45 @@
 package subprocess
 
 import (
+	"io"
 	"io/ioutil"
+	"os"
 
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v2"
 )
 
 // NewProcess is a constructor for a process object. Represents a process with argument config. Returns an address to process
 func NewProcess(name string, args []string, stdoutPath string, stderrPath string) (*Process, error) {
+	var stdout, stderr io.WriteCloser
+	var err error
+	// Setup output capture.
+	if stdoutPath != "" {
+		stdout, err = os.Create(stdoutPath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "Unable to open stdout file")
+		}
+	}
+	if stderrPath == stdoutPath {
+		stderr = stdout
+	} else if stderrPath != "" {
+		stderr, err = os.Create(stderrPath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "Unable to open stderr file")
```

Comment:
Error wrapping and including file name same as stdoutPath above.

Reply:
Fixed.